package.json

```diff
@@ -54,7 +54,7 @@
     "prepublishOnly": "publish-please guard"
   },
   "dependencies": {
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.10.0",
```

Comment:
Few babel plugins (`@babel/plugin-proposal-class-properties`, `@babel/plugin-proposal-decorators`, and etc.) are released with the `@babel/core` module. So, let's update their versions too.

Update all `@babel/` plugins with `^7.0.0` version to `^7.12.1` one for the consistency reason.